### PR TITLE
feat: create rds-proxy

### PR DESCRIPTION
**Pull request by Kasarpooja**
## what
created rds proxy

### CODE REVIEW

File: _example/aurora-postgres/example.tf [hunk 1]
```diff
@@ -23,18 +25,37 @@ module "vpc" {
...
+  vpc_id              = module.vpc.vpc_id
+  type                = "public-private"
+  igw_id              = module.vpc.igw_id
+  cidr_block          = module.vpc.vpc_cidr_block
+  ipv6_cidr_block     = module.vpc.ipv6_cidr_block
+  enable_ipv6         = false
 }
 
+
+# module "private-subnets" {
```
Comment (h1manshu98) [inline _example/aurora-postgres/example.tf]: remove this commented part if not required.

File: _example/aurora-postgres/example.tf [hunk 2]
```diff
@@ -1,10 +1,12 @@
 provider "aws" {
-  region = "eu-north-1"
+  region = local.region
 }
 
 locals {
-  environment = "test"
   name        = "aurora-postgres"
+  environment = "test"
+  label_order = ["environment", "name"]
```
Comment (h1manshu98) [inline _example/aurora-postgres/example.tf:8]: use this `label_order` in all module being called.

File: outputs.tf [hunk 3]
```diff
@@ -158,11 +158,93 @@ output "db_parameter_group_id" {
   value       = try(aws_db_parameter_group.this[0].id, null)
 }
 
-################################################################################
-# CloudWatch Log Group
-################################################################################
+#######################################################################################################################
+# RDS-PROXY: proxy output will show outputs other than empty string `""` , only when `create_db_proxy` is set to `true`
```
Comment (archit-cd) [inline outputs.tf:162]: what's this comment for?

Comment (h1manshu98) [reply outputs.tf:162]: The RDS Proxy is an optional resource. The default value of `create_db_proxy` to create RDS Proxy is set to `false`
Hence the outputs for RDS Proxy will show values other than `null` or `" "` only when `create_db_proxy` is set to `true`

**Review by archit-cd — APPROVED** [review]
LGTM

**Review by pendharevaibhav-cd — APPROVED** [review]
LGTM!

**Review by anmolnagpal — APPROVED** [review]

**Pull request merged**